src/features/user/RegisterTrees/RegisterTreesWidget.tsx

```diff
@@ -422,7 +422,7 @@ type FormData = {
 
 export default function RegisterTreesWidget() {
   const { user, token } = useUserProps();
-  const { t, ready } = useTranslation(['me', 'common']);
+  const { ready } = useTranslation(['me', 'common']);
```

Comment:
It does not appear like we need `ready` either, if we are not using translations in any way. Please remove if not needed.